presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanNodeStats.java

```diff
@@ -41,6 +42,7 @@
 
     private final Map<String, OperatorInputStats> operatorInputStats;
     private final Map<String, OperatorHashCollisionsStats> operatorHashCollisionsStats;
+    private final Optional<WindowNodeStats> windowNodeStats;
```

Comment:
Not a fan of this direction - this is making PlanNodeStats more and more of a union structure. OperatorHashCollisionsStats is a little bit better because it applies to the couple of operators that build a hashmap.

Can we somehow move this in a more general direction? We could use inheritance, or maybe we could add printing support to OperatorInfo?

Need to discuss this with @martint, but I would like us to at least consider other ideas rather than making this class more complex.